src/web/app/pages-instructor/instructor-student-records-page/instructor-student-records-page.component.ts

```diff
@@ -56,21 +51,32 @@ export class InstructorStudentRecordsPageComponent implements OnInit {
    * Loads the student's records based on the given course ID and email.
    */
   loadStudentRecords(courseid: string, studentemail: string): void {
-    const paramMap: { [key: string]: string } = { courseid, studentemail };
-    this.httpRequestService.get('/students/records', paramMap).subscribe((resp: StudentRecords) => {
-      this.courseId = resp.courseId;
-      this.studentName = resp.studentName;
-      this.studentEmail = resp.studentEmail;
-      this.studentProfile = resp.studentProfile;
-      this.sessions = resp.sessionNames.map((sessionName: string) => ({ name: sessionName, isCollapsed: false }));
+    this.studentProfileService.getStudentProfile(studentemail, courseid).subscribe((resp: StudentProfile) => {
+      this.courseId = courseid;
+      this.studentName = resp.name;
+      this.studentEmail = resp.email;
+      this.studentProfile = resp;
       if (!this.studentProfile) {
         this.statusMessageService.showWarningMessage(
-                'Normally, we would show the student\'s profile here. '
-                + 'However, either this student has not created a profile yet, '
-                + 'or you do not have access to view this student\'s profile.');
+            'Normally, we would show the student\'s profile here. '
+            + 'However, either this student has not created a profile yet, '
+            + 'or you do not have access to view this student\'s profile.');
       }
     }, (resp: ErrorMessageOutput) => {
       this.statusMessageService.showErrorMessage(resp.error.message);
     });
+
+    this.feedbackSessionsService.getFeedbackSessionsForInstructor(courseid).subscribe((fss: FeedbackSessions) => {
```

Comment:
The call is for? This is student's record page which has nothing to do with the feedbackSessions that instructors need to answer.